src/main/java/org/eclipse/yasson/model/PropertyModel.java

```diff
@@ -155,21 +156,25 @@ private void introspectNumberFormatter(Property property, AnnotationIntrospector
          * If @JsonbNumberFormat is placed on getter implementation must use this format on serialization.
          * If @JsonbNumberFormat is placed on setter implementation must use this format on deserialization.
          * If @JsonbNumberFormat is placed on field implementation must use this format on serialization and deserialization.
+         *
+         * Priority from high to low is getter / settrer > field > class > package > global configuration
          */
         Map<AnnotationTarget, JsonbNumberFormatter> jsonNumberFormatCategorized = introspector.getJsonNumberFormatter(property);
```

Comment:
I realized there is missing PACKAGE level in AnnotationTarget, which also applies to number format, but I think it is good to go now, I will add it later.